simplified-app-shared/src/main/res/values-v21/styles.xml

```diff
@@ -284,6 +284,20 @@
 
   </style>
 
+  <style
```

Comment:
Could you add comments to these two entries saying this is for Porterville? I see you have them in the other styles file already 👍